spdx/relationship.py

```diff
@@ -101,18 +102,15 @@ def validate(self, messages):
         """Returns True if all the fields are valid.
         Appends any error messages to messages parameter.
         """
-        messages = self.validate_relationship(messages)
-
-        return messages
+        return self.validate_relationship(messages)
 
     def validate_relationship(self, messages):
-        messages = messages if messages is not None else []
         r_type = self.relationship.split(" ")[1]
         if r_type not in [name for name, _ in RelationshipType.__members__.items()]:
-            messages = messages + [
+            messages.append(
                 "Relationship type must be one of the constants defined in "
                 "class spdx.relationship.Relationship"
-            ]
-            return messages
+            )
+            return False
```

Comment:
Could we instead always return ErrorMessages? here and elsewhere

Reply:
The docstring is different for that task, we are supposed to return True if the relationship is valid. I'll fix and change docstrings